releasenotes/notes/profiling-python-3.11-40056cf7a4313cec.yaml

```
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    profiler: add support for Python 3.11.
```

```suggestion
    profiling: add support for Python 3.11.
```

With the new release note guidelines we should use "profiling" instead of profiler: https://github.com/DataDog/dd-trace-py/pull/4551/files